### PR TITLE
Enable test regression validation

### PR DESCRIPTION
**Pull request by juwara0**
### This project uses [semver](semver.org), please check the scope of this pr:

- [X] #none# - documentation fixes and/or test additions
- [ ] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG
* **Added** test regression validation

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
   },
   "scripts": {
     "build": "ember build",
+    "coverage": "pr-bumper check-coverage",
```
Comment (job13er) [inline package.json]: I find this a little misleading. `npm run coverage` sounds like you're running coverage, not checking it.

**Comment by job13er** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-list/pull-request/133/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-list)

File: package.json (hunk 1 above)
Comment (juwara0) [reply package.json]: I updated this command to `check-coverage`

**Comment by juwara0** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-list/pull-request/133/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-list)

**Pull request merged**